integration/test/upgradetiller/upgrade_tiller_test.go

```diff
@@ -36,7 +36,7 @@ func TestUpgradeTiller(t *testing.T) {
 			t.Fatalf("could not get tiller image %#v", err)
 		}
 		if tillerImage != currentTillerImage {
-			t.Fatalf("tiller image == %#q, want %#q", tillerImage, outdatedTillerImage)
+			t.Fatalf("tiller image == %#q, want %#q", tillerImage, currentTillerImage)
```

Comment:
Uncovered this (likely copy/paste) bug in error message - "current" was used for comparison, but "outdated" was used in the message.

Reply:
Good catch, thx!